.gitignore

```diff
@@ -0,0 +1,3 @@
+
+queue_it.queue_token.v1-0.1.0.gem
```

Comment:
You should `*` the version number of the file if you're gonna to ignore it!

Reply:
Done 👍